enterprise/server/cmd/server/main.go

```diff
@@ -193,6 +193,13 @@ func main() {
 	if redisTarget := configurator.GetDefaultRedisTarget(); redisTarget != "" {
 		redisClient := redisutil.NewClient(redisTarget, healthChecker, "default_redis")
 		realEnv.SetDefaultRedisClient(redisClient)
+		maxValueSizeBytes := int64(0)
+		if redisConfig := configurator.GetDefaultRedisConfig(); redisConfig != nil {
+			maxValueSizeBytes = redisConfig.MaxValueSizeBytes
+		}
```

Comment:
The `maxValueSizeBytes` is only needed by the cache and is unused by the implementations of `MetricsCollector` and `KeyValStore`. I think we can hard-code this as zero for now, to avoid needing to introduce a `RedisConfig` section for the default Redis target, and also add a TODO to move the MetricsCollector and KeyValStore implementations out of `redis_cache.go` and into their own packages.

Reply:
Sounds good.